README.md

```diff
@@ -341,6 +341,22 @@ Hence, we recommend looking at both metrics together. Branch coverage might also
 overall metric to look at - while you might be missing only 10% of your lines that might
 account for 50% of your branches for instance.
 
+## Primary Coverage
```

Comment:
Let me know if this should be rephrased or go in a different place in the README!

Reply:
seems good to me, but README reordering I often rather do in a whole as it's a bit hard to do in PRs - as long as content is good to go I can put it where I see fit :D